src/installer/installer.go

```diff
@@ -716,7 +716,7 @@ func (i *installer) updateReadyMasters(nodes *v1.NodeList, readyMasters *[]strin
 
 			host, ok := common.HostMatchByNameOrIPAddress(node, inventoryHostsMap, knownIpAddresses)
 			if !ok {
-				return fmt.Errorf("Node %s is not in inventory hosts", node.Name)
+				return fmt.Errorf("node %s is not in inventory hosts", node.Name)
```

Comment:
PRs that are backport candidates should strive for minimal changes, so it's best to avoid small irrelevant cleanups such as this

Reply:
right.